indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -50,6 +43,40 @@
                                    data-title="{% trans %}Delete template{% endtrans %}"
                                    data-update="#email-template-manager">
                                 </a>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="i-box-content">
+                    {% trans %}This email will be sent when the abstract is:{% endtrans %}
+                        <ul>
+                            {% for rule in tpl.rules %}
+                                {% if rule %}
+                                    <li class="notification-rule">
+                                        {{ states[rule['state'][0]] }}
+                                        {% if 'track' in rule %}
+                                            <strong>in</strong>
+                                            {% if rule['track'][0] is number %}
+                                                {{ track[rule['track'][0]].title }}
+                                            {% else %}
+                                                {{ 'any track' if rule['track'][0] == '*' else 'no track' }}
+                                            {% endif %}
+                                        {% endif %}
+                                        {% if 'contribution_type' in rule %}
+                                            <strong>as</strong>
+                                            {% if rule['contribution_type'][0] is number %}
+                                                {{ contr_type[rule['contribution_type'][0]].name }}
+                                            {% else %}
+                                                {{ 'any type' if rule['contribution_type'][0] == '*' else 'no type' }}
```

Comment:
Yes. And I'd do the any/no type checks first and just use an `else` block for the last one (assuming you can easily detect "no type" without checking whether it's not a number, e.g. `is none`)